py2hwsw/py2hwsw_document/document/tsrc/special_cases.tex

```diff
@@ -0,0 +1,15 @@
+% SPDX-FileCopyrightText: 2024 IObundle
+%
+% SPDX-License-Identifier: MIT
+
+Most of the cores provived by the py2hwsw's library are built using the standard interfaces mentioned in section~\ref{sec:py2_standard_interfaces}.
+
+However, there are some cores that due to limitations of the standard interfaces, rely instead on internal py2hwsw methods for extra features.
+The following list describes the cores don't rely solely on the standard interfaces.
+
+\begin{itemize}
+\item \textbf{iob\_system}: This core uses the `is\_system` attribute to enable an internal py2hwsw method that automatically fixes the address widths of the cbus interfaces of the system's peripherals.
+\item \textbf{iob\_csrs}: The py2hwsw tool contains an internal method to automatically search for the "iob\_csrs" subblock and insert a "cbus\_s" port (if it doesn't exist yet) on the instantiator core of this subblock. It then connects this newly created "cbus\_s" port of the instantiator core to the iob\_csrs "control\_if\_s" port.
```

Comment:
This is the problematic part: it can't exist. If you want such a port, you must remove the iob_csr instance and do everything by hand.

Reply:
Resolved in 94e3ec1